accession/admin_views.py

```diff
@@ -86,22 +55,34 @@ def print_view(request, app_label, model_name, object_id):
                     else:
                         value = entry.__unicode__().encode("utf-8")
                         break
-        #if it isn't a related field
+        # if it isn't a related field
         else:
-            #get the value of the field
+            # get the value of the field
             if isinstance(row.__dict__[field.name], basestring):
                 value = row.__dict__[field.name].encode("utf-8")
             else:
                 value = row.__dict__[field.name]
         field_list.append({'label': field.verbose_name, 'value': value})
-    
+
     return render_to_response(
         "admin/accession/print_view.html",
         {
-        'requested_obj' : row,
-        'app_name' : app_label.title(),
-        'model_name': model_name.title(),
-        'field_list': field_list,
+            'requested_obj': row,
+            'app_name': app_label.title(),
+            'model_name': model_name.title(),
+            'field_list': field_list,
         },
         RequestContext(request, {}),
     )
+
+
+def export_csv(request, app, model):
+    try:
+        model = models.get_model(app, model)
+        response = render_to_csv_response(model.objects.all())
+    except AttributeError as e:
+        raise Http404(str(e))
+    except LookupError as e:
+        raise Http404(str(e))
```

Comment:
Should we be reusing the exception message? That might not be very user friendly. What if we had something a little more generic?

Also, because we are performing the same action for both the `AttributeError` and the `LookupError` we can consolidate this to 

```python
try:
# ...
except (AttributeError, LookupError) as e:
    # do stuff
```